index.js

```diff
@@ -0,0 +1,118 @@
+const MAX_DELAY = 3000;
+
+const inputElement = document.getElementById('polish');
+const formElement = document.getElementById('form');
+const stepsElement = document.getElementById('steps');
+const resultElement = document.getElementById('result');
+
+const rnd = () => Math.floor(Math.random() * MAX_DELAY + 1);
+
+const plus = (x, y) => {
+  return new Promise(resolve => {
+    setTimeout(() => resolve(x + y), rnd());
+  })
+};
+
+const minus = (x, y) => {
+  return new Promise(resolve => {
+    setTimeout(() => resolve(x - y), rnd());
+  })
+};
+
+const multiply = (x, y) => {
+  return new Promise(resolve => {
+    setTimeout(() => resolve(x * y), rnd());
+  })
+};
+
+const devide = (x, y) => {
+  return new Promise(resolve => {
+    setTimeout(() => resolve(x / y), rnd());
+  })
+};
+
+const showStep = (data) => {
+  const pElement = document.createElement('p');
+  pElement.className='card-text';
+  pElement.textContent=JSON.stringify(data);
+  stepsElement.appendChild(pElement);
+}
+
+const clearSteps = () => {
+  stepsElement.innerHTML = '';
+}
+
+const applyOperator = async (x, y, operator) => {
+  let res = 0;
+  switch (operator) {
+    case '+':
+      res = await plus(x, y);
+      break;
+    case '-':
+      res = await minus(x, y);
+      break;
+    case '*':
+      res = await multiply(x, y);
+      break;
+    case '/':
+      res = await devide(x, y);
+      break;
+    default:
+      break;
+  }
+  return res;
+}
+
+const isOperator = (element) => {
+  if ((element === '+') || (element === '*') || (element === '/') || (element === '-')) {
+    return true;
+  }
+  return false;
```

Comment:
you don't need use if here. You can return (element === '+') || (element === '*') || (element === '/') || (element === '-') and it will convert to boolean value

Reply:
Done.